numpyro/mcmc.py

```diff
@@ -77,6 +79,12 @@ def get_diagnostics_str(hmc_state):
                                                               hmc_state.mean_accept_prob)
 
 
+def get_progbar_desc_str(num_warmup, hmc_state):
+    if hmc_state.i < num_warmup:
```

Comment:
I verified that the performance is unimpacted, there was a problem with the logistic_regression test where we weren't passing in the `algo` argument so it was actually running NUTS instead of HMC and appeared to be faster.

Reply:
haha, this is truly tricky to detect.